build.gradle

```diff
@@ -71,6 +71,7 @@ ext {
     androidxNavigationVersion = '2.4.1'
     androidxPreferenceVersion = '1.2.0'
     androidxRecyclerviewVersion = '1.2.1'
+    androidxSecurityVersion = '1.1.0-alpha03'
```

Comment:
Is there a non-alpha version that works?

Reply:
No, alpha is needed to support API 21.